src/components/auth/Oauth/ClientCredentials/ClientCredentials.tsx

```diff
@@ -19,12 +15,5 @@ interface ClientCredsFlowProps {
   setSelectedConnection: (connection: Connection | null) => void;
 }
 export function ClientCredentials({ ...props }: ClientCredsFlowProps) {
-  if (isChakraRemoved) {
-    return <ClientCredsContainer {...props} />;
-  }
-
-  if (props.explicitWorkspaceRequired) {
-    return (<WorkspaceOauthClientCredsFlow {...props} />);
-  }
-  return (<NoWorkspaceOauthClientCredsFlow {...props} />);
+  return <ClientCredsContainer {...props} />;
```

Comment:
ClientCredsContainer was generalized in the refactor to no longer need separate flows for workspace (will dynamically add fields into the flow)